querybookchecks.js

```diff
@@ -0,0 +1,202 @@
+const Pool = require('pg').Pool
```

Comment:
please remove Pool reference because we don't need it anymore

Reply:
I fixed it